BOJ/solved.ac_class/Class02/10828.스택/sAp00n.py

```diff
@@ -0,0 +1,61 @@
+from sys import stdin
+
+
+class Stack:
+    def __init__(self):
+        self.stack_data = []
+
+    def push(self, data_in):
+        self.stack_data.append(data_in)
+
+    def pop(self):
+        if self.empty():
+            print(-1)
+        else:
+            temp = self.stack_data[-1]
+            self.stack_data = self.stack_data[:-1]
+            print(temp)
+
+    def size(self):
+        print(len(self.stack_data))
+
+    def empty(self):
+        if len(self.stack_data) == 0:
+            return True
+        else:
+            return False
```

Comment:
```suggestion
        return len(self.stack_data) == 0
```

boolean은 바로 반환해도 됩니다!